### DISCUSSION TITLE
Google APIs Audit-Recording

### DISCUSSION THREAD

**Discussion by maxoux53** [badmojr/1Hosts#554] [Q&A] [answered]
<br>
<h2 align="center">Submit Form</h2>
<div align="center">
  Get your issue resolved quickly! Fill in the form accurately.
</div>

**Lists in use:**
- [ ] mini
- [ ] Lite
- [x] Pro
- [ ] Xtra
<br>

**Client:**
NextDNS

<br>

**Domain:**
```
auditrecording-pa.googleapis.com
```
<br>

**Details:**
What is auditrecording-pa.googleapis? It's apparently an audit recording domain...
Shouldn't it be blocked?

<div align="center">
  <h2>Thank you for supporting 1Hosts.</h2>
  <sub>It’s people like you who make these lists great! ❤<ub>
</div>

**Comment by badmojr** [comment]
Not unless someone  presents thorough evidence on why it should be.
You want it blocked? The onus is on you to present such detailed info.

**Comment by maxoux53** [comment]
> Not unless someone presents thorough evidence on why it should be. You want it blocked? The onus is on you to present such detailed info.

I know, which is why I said "What is auditrecording-pa.googleapis? (...) Shouldn't it be blocked?" these are questions.
I ask your opinion and that of the others.

**Comment by maxoux53** [comment]
You should add the Discussion section in this repo, it would be convenient I think

**Comment by badmojr** [comment]
> add the Discussion section in this repo, it would be convenient I think

Done!

**Comment by maxoux53** [comment, marked as answer]
Some news about this:
Since the Android 12 update, auditrecording-pa has been requested 80 times (whereas before the update it had never been requested).
So it's certainly a new android tracking domain because in less than 8 days (I updated my phone the 6th of May and now we are the 14th) it did 80 requests so 10 requests per day on average.
Maybe we'll find some informations about it next weeks...
